src/frontend/screens/Login/index.css

```
@@ -16,61 +18,171 @@
 }
 
 .loginContentWrapper {
-  width: 100%;
+  width: 75%;
   height: 100%;
   display: flex;
-  align-items: center;
   justify-content: center;
   flex-direction: column;
   grid-column: 1;
   grid-row: 1;
   z-index: 1;
+  margin: 0 auto;
 }
 
 .runnerList {
   display: flex;
-  flex-direction: row;
+  flex-direction: column;
   align-items: center;
   justify-content: center;
   position: relative;
   user-select: none;
-  width: 100%;
   flex-wrap: wrap;
+  background-color: var(--background);
+  padding: var(--wrapper-padding);
+  border-radius: 10px;
+  margin-top: 25%;
+  min-width: 300px;
+}
+
+.languageSelector {
+  position: absolute;
+  bottom: calc(100% + var(--login-padding));
+  left: 0;
+  width: 300px;
+}
+
+.runnerList,
+.languageSelector {
+  width: 100%;
 }
 
 .goToLibrary {
   position: relative;
   font-family: var(--font-primary-bold);
-  top: 10%;
-  color: var(--text-default);
-  background-color: var(--background-darker);
+  font-weight: 700;
+  color: var(--background);
+  background-color: var(--accent);
   border: none;
-  font-size: var(--text-xl);
-  padding: var(--space-md);
+  padding: 10px;
+  padding-left: 40px;
+  padding-right: 40px;
```

We can use the shorthand padding to set all of them at once:

```suggestion
  padding: 10px 40px;
```